src/utils/sidebar.js

```diff
@@ -111,9 +102,9 @@ function normalizeSidebar({
 
   data.forEach(rawItem => {
     const isShortcut = typeof rawItem === 'string'
-    const item = isShortcut ? { slug: rawItem } : rawItem
+    rawItem = isShortcut ? { slug: rawItem } : rawItem
```

Comment:
Agree in general but this is just an anonymous arrow fn for the `forEach` block. Should I revert anyway?

Reply:
p.s. similar to how it's done here by necessity: 
https://github.com/iterative/dvc.org/blob/dd6723cc01b7fb88de8a31bd3b0b55c3067d4a76/src/Documentation/RightPanel/RightPanel.js#L19-L22
https://github.com/iterative/dvc.org/blob/dd6723cc01b7fb88de8a31bd3b0b55c3067d4a76/src/Documentation/RightPanel/RightPanel.js#L34-L37
https://github.com/iterative/dvc.org/blob/dd6723cc01b7fb88de8a31bd3b0b55c3067d4a76/src/styles.js#L54-L58

Reply:
I did find a place where it's probably not justified though:
https://github.com/iterative/dvc.org/blob/dd6723cc01b7fb88de8a31bd3b0b55c3067d4a76/src/Documentation/Markdown/Markdown.js#L158-L164

Which ones should we rewrite?

Reply:
I'd say we should try to write immutable code everywhere we can by default and only mutate things if there is no other choice.

I'm ok with the images example because we didn't mutate images themselves and just call methods on them.

I'm more or less ok with accumulator example because we only use this arg in reduce and exactly to do this things.

But I'd rewrite `isInsideCodeBlock` code if there is a time for that later.

Reply:
Agreed. OK, let me try to rewrite the other 2 examples and then I'll merge this 🙂

Reply:
Done!
See https://github.com/iterative/dvc.org/pull/891/files?file-filters%5B%5D=.js#diff-b1d807de8934c24309015d3c1ea18882
and https://github.com/iterative/dvc.org/pull/891/files?file-filters%5B%5D=.js#diff-65c1b1a621d382085d912a379ad71c9b
.